<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qalaxy Quest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="aloita_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Uusi peli</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keskeyta_Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lopeta peli</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sulje_Button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>530</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sulje</string>
    </property>
   </widget>
   <widget class="QLabel" name="pelaaja_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Pelaaja</string>
    </property>
   </widget>
   <widget class="QLabel" name="nimi_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pisteet_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Pisteet</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pisteet_lcd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grafiikka_ikkuna">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>581</width>
      <height>500</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
   </widget>
   <widget class="QLabel" name="raha_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Raha</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="raha_lcd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="pelastetut1_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pelastetut </string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pelastetut2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>alukset</string>
    </property>
   </widget>
   <widget class="QLabel" name="menetetyt1_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Menetetyt</string>
    </property>
   </widget>
   <widget class="QLabel" name="menetetyt2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>alukset</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pelastetut_lcd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="menetetyt_lcd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tahtijarjestelma1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tähtijärjestelmä:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tahtijarjestelma_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>text</string>
    </property>
   </widget>
   <widget class="QLabel" name="populaatio1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Populaatio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="populaatio_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="elinkeino1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Elinkeino:</string>
    </property>
   </widget>
   <widget class="QLabel" name="elinkeino_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="liiku_Button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Liiku</string>
    </property>
   </widget>
   <widget class="QLabel" name="pelinTila_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="korjaa_Button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>230</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Korjaa</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>aloita_Button</zorder>
   <zorder>keskeyta_Button</zorder>
   <zorder>sulje_Button</zorder>
   <zorder>pelaaja_label</zorder>
   <zorder>nimi_label</zorder>
   <zorder>pisteet_label</zorder>
   <zorder>pisteet_lcd</zorder>
   <zorder>grafiikka_ikkuna</zorder>
   <zorder>raha_label</zorder>
   <zorder>raha_lcd</zorder>
   <zorder>pelastetut1_label</zorder>
   <zorder>pelastetut2_label</zorder>
   <zorder>menetetyt1_label</zorder>
   <zorder>menetetyt2_label</zorder>
   <zorder>pelastetut_lcd</zorder>
   <zorder>menetetyt_lcd</zorder>
   <zorder>tahtijarjestelma1_label</zorder>
   <zorder>tahtijarjestelma_label</zorder>
   <zorder>populaatio1_label</zorder>
   <zorder>populaatio_label</zorder>
   <zorder>elinkeino1_label</zorder>
   <zorder>elinkeino_label</zorder>
   <zorder>liiku_Button</zorder>
   <zorder>pelinTila_label</zorder>
   <zorder>korjaa_Button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sulje_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
